### PR TITLE
Fix popup in editor

### PR DESCRIPTION
**Pull request by crazypsycho**

### CODE REVIEW

File: trunk/Public/DynamicConditionsPublic.php [hunk 1]
```diff
@@ -248,10 +248,12 @@ private function convertAcfDate( $id, array $data ) {
      * @param Locations_Manager $locationManager
      */
     public function checkPopupsCondition( $locationManager ) {
-        if ( ! empty( Plugin::$instance->preview ) && Plugin::$instance->preview->is_preview_mode() ) {
+        if ( !empty( Plugin::$instance->preview ) && Plugin::$instance->preview->is_preview_mode() ) {
+            return;
+        }
+        if ( !empty( Plugin::$instance->editor ) && Plugin::$instance->editor->is_edit_mode() ) {
```
Comment (mheavrin) [inline trunk/Public/DynamicConditionsPublic.php]: Two `if` statements right after another that do the same thing? I'd prefer combining the conditions.

Comment (crazypsycho) [reply trunk/Public/DynamicConditionsPublic.php]: I think with two conditions, it is more readable

Comment (mheavrin) [reply trunk/Public/DynamicConditionsPublic.php]: You could always put the conditions in separate methods?

Comment (crazypsycho) [reply trunk/Public/DynamicConditionsPublic.php]: Done

**Pull request merged**